### PR TITLE
Feat/Side Navigation Menu

### PR DESCRIPTION
**Pull request by akshay-codemonk**
UI updates
- Add pagination for tables
- Add side navigation menu
- Load logo from static dir
- Dynamically load profile display icon
- Overall CSS code refactor

Fix workspace deletion
- Delete the schedule on workspace delete

Screenshots
- Side Navigation menu

Screenshot from 2019-10-11 13-35-32

- Increase spacing between text fields
Screenshot from 2019-10-11 13-39-01

- Pagination and error message
Screenshot from 2019-10-11 13-39-44

- Schdule page
Screenshot from 2019-10-11 13-39-18

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ COPY . /code/
 WORKDIR /code
 
 # Run lint checks
-RUN pylint --load-plugins pylint_django apps/**/ --disable=missing-docstring --max-line-length=120
+RUN pylint --load-plugins pylint_django apps/**/ --disable=missing-docstring --disable=unused-variable --disable=unused-argument --max-line-length=120
```
Comment (snarayanank2) [inline Dockerfile]: why did you disable the unused variable and unused arg checks.. they're useful

File: apps/sync_activity/utils.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    :param connection: Fyle connection
+    :return: file_id
+    """
+    file_id = connection.Files.post(file_name)['id']
+    s3_upload_url = connection.Files.create_upload_url(file_id)['url']
+    connection.Files.upload_file_to_aws('application/x-sqlite3', file, s3_upload_url)
+    return file_id
+
+
+def update_activity_status(activity, message, status):
```
Comment (snarayanank2) [inline apps/sync_activity/utils.py]: why is this not in the model class?

File: apps/xero_workspace/tasks.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+from apps.sync_activity.models import Activity
+from apps.sync_activity.utils import upload_sqlite, update_activity_status
+from apps.xero_workspace.models import XeroCredential, FyleCredential, Workspace, WorkspaceActivity
+from apps.xero_workspace.utils import connect_to_xero, connect_to_fyle, generate_invoice_request_data, create_invoice, \
+    load_mapping, transform
+
+
+def sync_xero(workspace_id, activity_id):
+    """
+    Synchronise data from Fyle to Xero
```
Comment (snarayanank2) [inline apps/xero_workspace/tasks.py]: can all the business logic reside in the model? the tasks.py can simply fetch the model and call the function. that way you can share the business logic between the view and background task

Comment (akshay-codemonk) [reply apps/xero_workspace/tasks.py]: The synchonization depends on two models workspace and activity, I  am not sure wheather it is right to put it under the workspace model

File: Dockerfile (hunk 1 above)
Comment (akshay-codemonk) [reply Dockerfile]: - Disabled unused-variables due to tuple unpacking, where only one variable was used
- Disabled unused-arguments due to django signal method, where kwargs is a required but unused argument

this is fixed in 21a8757

**Pull request merged**